apps/address-service/domains/common/utils/services/InjectionsSeeker.js

```diff
@@ -0,0 +1,190 @@
+const { AddressInjection } = require('@address-service/domains/address/utils/serverSchema')
+const get = require('lodash/get')
+const { INJECTIONS_PROVIDER } = require('@address-service/domains/common/constants/providers')
+
+const SPECIAL_SYMBOLS_REGEX = /[!@#$%^&*)(+=.\-_:;"'`[\]]/g
+
+class InjectionsSeeker {
```

Comment:
Could you please drop a few lines about what this class does

Reply:
Done